src/azure-cli/azure/cli/command_modules/backup/commands.py

```diff
@@ -34,45 +38,53 @@ def load_command_table(self, _):
         g.custom_command('backup-properties set', 'set_backup_properties', client_factory=backup_storage_configs_cf)
         g.custom_command('delete', 'delete_vault', confirmation=True)
 
-    with self.command_group('backup container', backup_custom, client_factory=backup_protection_containers_cf) as g:
-        g.show_command('show', 'show_container')
-        g.command('list', 'list_containers', table_transformer=transform_container_list)
+    with self.command_group('backup container', backup_custom_base, client_factory=protection_containers_cf) as g:
```

Comment:
It looks  backup_custom_base is still a custom type, right?

Reply:
I have updated custom_command type for the newly added commands. For the existing commands i have not made any changes. Those changes can be discussed later.